spec/aws/vpc.go

```diff
@@ -4,4 +4,8 @@ type VPC struct {
 	CIDR              string `json:"cidr" yaml:"cidr"`
 	PrivateSubnetCIDR string `json:"privateSubnetCidr" yaml:"privateSubnetCidr"`
 	PublicSubnetCIDR  string `json:"publicSubnetCidr" yaml:"publicSubnetCidr"`
+
+	// PeerID is the ID of the VPC which we should peer to.
+	// e.g: the vpc of the host cluster.
+	PeerID string `json:"peerId" yaml:"peerId"`
```

Comment:
I am not too much into the use cases here. Would it make sense to have a list of peer IDs to peer with?

Reply:
Uncertain. This is the vpc of the host cluster we want the guest vpcs to peer with. I don't see how we could have multiple vpcs for the host cluster.